simple-search/service.py

```diff
@@ -0,0 +1,44 @@
+from authority import API_KEY
+from json import dumps, loads
+from logging import error
+from requests import get
+
+
+class Endpoint:
+    SEARCH = 'https://openapi.companybooknetworking.com/1.0/search/search?'
+
+
+HEADERS = {
+    'Accept': 'application/json',
+    'x-api-key': API_KEY
+}
+
+
+def format_result(result):
+    return {
+        'url': 'https://www.companybooknetworking.com/{}'.format(result['company_id']),
+        'name': result['name'],
+        'snippet': result['query_snippet'] if 'query_snippet' in result else ''
+    }
+
+
+def format_response(raw_result):
+    parsed_result = loads(raw_result)
+    result = parsed_result['result']
+    docs = result['docs']
+    resultset = [format_result(doc) for doc in docs]
+    return resultset
+
+
+def search(query, country=''):
+    parameters = {}
+    if query:
+        parameters['q'] = query
+    if country:
+        parameters['country'] = country
```

Comment:
Or actually. No. Why? Default PyCharm formatting would not add a newline. The function is not long. I don't see a newline making sense there.